.github/workflows/build.yml

```diff
@@ -236,13 +236,8 @@ jobs:
           arch: ${{ matrix.arch }}
           instructions: |
             make image
-          # The ECR tag used below is to allow us to test by pushing to a private ECR registry.
```

Comment:
Long term, I think we'll move builds to `goreleaser` like (most) of the rest of OpenBao. I don't know that I see immediate value in us doing actual _releases_; when we stabilize it and land the features we want, maybe somebody would be willing to take on that work.

But in the meantime, I'm also happy doing _a_ build to prove it works (even `make bin`), but we don't need it on all platforms.

Reply:
Even if we get to the point of wanting to do releases, I'm not sure I see value in supporting all of these OS/Arch options since this can be run from other system (not the server) just fine and would likely be a more representative benchmark.